doc/cascadia/profiles.schema.json

```
@@ -54,6 +54,7 @@
         "scrollUpPage",
         "splitPane",
         "switchToTab",
+        "toggleBorderless",
```

name changed in code

```suggestion
        "toggleFocusMode",
```